asistente_ladm_col/core/ili2db.py

```diff
@@ -424,6 +441,8 @@ def validate(self, db, model_names=list(), xtflog_path='', configfile_path=''):
 
     def on_process_started(self, command):
         self._log += command + '\n'
+        self.process_started.emit(command)
```

Comment:
Just for the record, you can also chain two signals, instead of emitting the second one on your own. That is,

    importer.process_started.connect(self.process_started)

Don't have a preferred way, though, so let's keep it as it is.